Selin_cars.py

```diff
@@ -0,0 +1,63 @@
+import random
+
+
+class Car:
+    def __init__(self, model: str, color: str, economy: int):
+        self.model = model
+        self.color = color
+        self.economy = economy
+        self.mileage = 0
+        self.fuel = 100
+
+    def drive(self, distance):
+        fuel = distance / self.economy
+        if fuel <= self.fuel:
+            self.mileage += distance
+            self.fuel -= fuel
+        else:
+            print("Not enough fuel")
+
+    def fuel_up(self):
+        self.fuel = min(self.fuel + 20, 100, 200)

```

Comment:
щось трохи перемудрив тут.
Достатньо було:
```suggestion
        self.fuel += 20

```